database-migration/src/test/java/uk/ac/ebi/eva/dbmigration/mongodb/ExtractStatisticsFromVariantTest.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbmigration.mongodb;
+
+import com.github.fakemongo.Fongo;
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import com.mongodb.util.JSON;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test {@link ExtractStatisticsFromVariant}
+ */
+public class ExtractStatisticsFromVariantTest {
```

Comment:
Please add methods to test:
* `migrateStatistics` and `removeStatisticsFromVariants` separately
* That all the migrated statistics are linked to an existing variant ID from the variants collection